presets.js

```diff
@@ -0,0 +1,416 @@
+const { combineRgb } = require('@companion-module/base')
+
+const CHOICES_SOURCES = [
```

Comment:
You define the same choices at different places, wouldn't it be better to do it in one place, e.g. choices.js